sms-authenticator/src/main/java/netzbegruenung/keycloak/authenticator/gateway/ApiSmsService.java

```diff
@@ -100,31 +101,31 @@ public void send(String phoneNumber, String message) {
 			} else {
 				logger.errorf("Failed to send message to %s [%s]. Validate your config.", phoneNumber, payload);
 			}
-		} catch (Exception e){
+		} catch (Exception e) {
 			logger.errorf(e, "Failed to send message to %s with request: %s. Validate your config.", phoneNumber, request != null ? request.toString() : "null");
 		}
 	}
 
 	public Builder json_request(String phoneNumber, String message) {
 		String sendJson = "{"
-			.concat(apitokenattribute != "" ? String.format("\"%s\":\"%s\",", apitokenattribute, apitoken): "")
-			.concat(String.format("\"%s\":\"%s\",", messageattribute, message))
-			.concat(String.format("\"%s\":%s,", receiverattribute, String.format(receiverJsonTemplate, phoneNumber)))
-			.concat(String.format("\"%s\":\"%s\"", senderattribute, senderId))
-			.concat("}");
+						  + Optional.ofNullable(apitokenattribute).map(it -> String.format("\"%s\":\"%s\",", it, apitoken)).orElse("")
```

Comment:
Using String.concat is the same as using the + operator but is more standard in the Java world